src/imageViewer/webpackModules/imageViewer.tsx

```diff
@@ -89,8 +89,8 @@ export default function ImageViewer({
     (e: MouseEvent) => {
       if (!dragging) return;
 
-      setX((prevX) => prevX + e.movementX / zoom);
-      setY((prevY) => prevY + e.movementY / zoom);
+      setX((prevX) => prevX + e.movementX / (zoom * 2.0));
```

Comment:
I think there might be some DPI scaling shenanigans going on, because it pans fine for me - is your Discord set to 200% size?

Reply:
OH THAT EXPLAINS THE EXACT NUMBER yeah i was panning and it was moving exactly twice as much as i was scrolling, I ran into something similar when I was working on a personal project so i figured it was that, i'll put in a more reasonable fix lmfao, if it wasn't exactly 2 I wouldve assumed scaling